conformance/fhir-ig-carin-bb/src/test/resources/JSON/StructureDefinition-C4BB-Patient.json

```diff
@@ -0,0 +1,5908 @@
+{
```

Comment:
I think I'd have preferred a minimal example that shows the issue.  Will we remove it once we package the next version of C4BB?

Reply:
A minimal example is hard to get right so I decided to use one that I knew allowed us to exercise what we needed. Yes, we will remove it when we upgrade.